music21/musicxml/m21ToXml.py

```diff
@@ -37,6 +37,7 @@
 from music21 import clef
 from music21 import chord
 from music21 import common
+from music21.common.types import OffsetQLIn
```

Comment:
Should be OffsetQL (not OffsetQLIn)  -- OffsetQLIn can be int, float, Fraction.  OffsetQL is just float, Fraction (the "In" means we're taking In external data and someone might provide an int as a setter, but we always use OffsetQL internally.).

Reply:
Done.